cmd/corectl/main.go

```diff
@@ -8,33 +8,38 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"path/filepath"
 	"strconv"
 	"time"
 
 	"chain/core/accesstoken"
 	"chain/core/config"
 	"chain/core/migrate"
 	"chain/crypto/ed25519"
+	"chain/database/raft"
 	"chain/database/sql"
-	chainjson "chain/encoding/json"
 	"chain/env"
 	"chain/log"
+	"chain/protocol/bc"
 	_ "chain/protocol/tx" // for BlockHeaderHashFunc
 )
 
 const version = "1.1.0"
 
 // config vars
 var (
-	dbURL = env.String("DATABASE_URL", "postgres:///core?sslmode=disable")
+	dbURL      = env.String("DATABASE_URL", "postgres:///core?sslmode=disable")
+	listenAddr = env.String("LISTEN", ":1999")
+	dir        = env.String("DIR", defaultDir())
```

Comment:
I'm going with `CORED_DATA_DIR` because we seem to like to stick nouns on the end of our env vars